common-utils/network/test-fixture-handler.js

```diff
@@ -0,0 +1,92 @@
+export class TestFixtureHandler {
```

Comment:
why one this class starts with Test?

Reply:
It is ported from Monterey. My interpretation is to emphasize handling test data while the other utils are for general interactions with page objects.